chainer/initializers/__init__.py

```diff
@@ -26,8 +26,13 @@
 from chainer import types  # NOQA
 
 
-def generate_array(initializer, shape, xp, dtype=None, device=None):
-    # type: (types.AbstractInitializer, types.ShapeSpec, types.Xp, types.DTypeSpec, types.DeviceSpec) -> types.NdArray  # NOQA
+def generate_array(
+        initializer: types.AbstractInitializer,
+        shape: types.ShapeSpec,
+        xp: types.Xp,
+        dtype: types.DTypeSpec = None,
+        device: types.DeviceSpec = None
```

Comment:
Optional?

Reply:
You're right, updated. (I confused with DTypeSpec.)